components/cd-button/cd-button.css

```diff
@@ -41,42 +42,51 @@
 }
 
 /* Default button */
-.cd-button {
+.cd-button,
+input[type='submit'] {
```

Comment:
I wonder if we should be applying this to stuff that isn't explicitly marked with `cd-button`. These selectors for `input` turn the component into more of a CSS reset, but its presence depends on one instance of the `cd-button` component being output by Drupal.

In its current form, it would apply styles unevenly if the page has `<input type="reset">` for example. I don't know if Drupal uses that markup, but HID does. No need to overhaul based on one edge case; It would be far easier to change HID. But I still wanted to highlight the possibility.

Reply:
Ya, good point. I think the input selector should probably be removed. IIRC `input[type='submit']` came from my trying to remove the left-over SASS components from https://github.com/UN-OCHA/common_design/blob/master/sass/components/ which are there for Drupal and not always used.
I'll review this in more detail and see how removing it affects Drupal and whether I need to add something additional elsewhere to account for this.